WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -123,13 +122,6 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
-    if (self.previousOrientation != UIInterfaceOrientationUnknown) {
-        if (UIInterfaceOrientationIsPortrait(self.previousOrientation) != UIInterfaceOrientationIsPortrait(self.interfaceOrientation)) {
-            [self.tableViewHandler refreshCachedRowHeightsForWidth:CGRectGetWidth(self.view.frame)];
```

Comment:
Ummmmmm i think this change would *definitely* bring side effects. Perhaps we'd just need to switch over...

```
- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id <UIViewControllerTransitionCoordinator>)coordinator
``` 

cc'ing @aerych!